django-stubs/contrib/contenttypes/prefetch.pyi

```diff
@@ -0,0 +1,9 @@
+from typing import Any
+
+from django.db.models import Prefetch
+from django.db.models.query import QuerySet
+
+class GenericPrefetch(Prefetch):
+    def __init__(self, lookup: str, querysets: list[QuerySet] | None = ..., to_attr: str | None = ...) -> None: ...
```

Comment:
Please, see https://github.com/django/django/pull/18125

Reply:
I've read the corresponding PR. It's a shame that if I had thought about it a little more, I would have found a bug. Still, thank you for letting me know the good opportunity :) I'm learning again today. If Django PR merges, I'll fix the code and apply it.

Reply:
I have confirmed that the code has been applied! I will fix it and commit it.